tests/loggen/loggen.c

```diff
@@ -284,11 +308,15 @@ start_plugins(GPtrArray *plugin_array)
       if (!plugin)
         continue;
 
-      if (plugin->start_plugin)
-        plugin->start_plugin((gpointer)&global_plugin_option);
+      if (plugin->start_plugin && plugin->is_plugin_activated())
+        {
+          plugin->start_plugin((gpointer)&global_plugin_option);
+          break;
+        }
     }
 
-  DEBUG("all plugin have been started\n");
+  DEBUG("%d plugin have been started\n", number_of_active_plugins);
```

Comment:
If we enforce `number_of_active_plugins == 1` [above](https://github.com/balabit/syslog-ng/pull/2074/files#diff-3fd3d9d588b133f7b4217b1ff775f55fR298), do we really need this DEBUG message? It will always print the same.

Reply:
deleted